.circleci/config.yml

```diff
@@ -56,6 +56,7 @@ jobs:
       - check-changed-files-or-halt:
           pattern: ^(packages)|(.js|jsx|ts|tsx|css|scss)$
       - yarn_install
+      - run: yarn run build
```

Comment:
This will run build again after `postinstall` right? should we build everything in `postInstall`?

Reply:
good point, I think we should simply delete automatic post-install

Reply:
but then locally, after `yarn` If the components are not built, I won't be able to use the package, `main` in package.json won't resolve because `dist` is not there, no?

Reply:
yes, we can make it clear in the documentation but I don't have a strong opinion